src/_ctf-private/ctf-sign-up-banner/CtfSignUpBanner.tsx

```diff
@@ -0,0 +1,34 @@
+import Link from 'next/link';
+
+import Logo from './CtfLogo.svg';
+
+import typewriter from 'analytics';
+
+const signUpLink =
+  'https://www.contentful.com/starter-templates/nextjs-blog/sign-up/?action=create_starter_template&template_name=blog';
+
+export const CtfSignUpBanner = () => {
+  const handleSignUpLinkClick = () => typewriter.signUpBannerInteracted({ ctaClicked: true });
+
+  return (
+    <div className="sticky top-0 z-[200] flex h-[54px] items-center justify-center gap-3 bg-[#0033A3]">
+      <p className="block font-systemUIFontFamilies text-sm text-colorWhite md:hidden">
+        Content managed via
+      </p>
+      <Logo />
+      <p className="hidden font-systemUIFontFamilies text-sm text-colorWhite md:block">
+        The content on this template is managed via Contentful
+      </p>
+      <Link href={signUpLink} target="_blank" onClick={handleSignUpLinkClick}>
+        <button className="upgrade-button rounded-[50px] bg-[#FFDA00] py-2 px-6 text-[#464E5B] hover:bg-[#EFC800]">
```

Comment:
can we set the height to 38px here? right now it's 36, just needs a lil bump up :)